backend/app/api/services/thread_service.py

```diff
@@ -0,0 +1,87 @@
+import uuid
+from typing import Optional
+
+from app.api.schemas import Filter, Message, MessageCreate, SupportCreate, SupportThread
+
+_threads: list[SupportThread] = []
+
+
+class SupportThreadService:
+    """
+    class for manipulation SupportThread entities.
+    Note: some functions should return exactly pydantic model!
+    """
+
+    @staticmethod
+    async def create(thread: SupportCreate) -> SupportThread:
+        """from function expected that it will store the thread in db"""
```

Comment:
"From function expected that" looks like a terrible part of the docs. Firstly, it is grammatically incorrect, secondly, this is already a docstring for the function, so you can explain what it does outright.

If I wanted a minimal yet useful docstring for this method, I would write something like:
```python
"""Create and return new `SupportThread` based on `SupportCreate`"""
```

Be sure to also fix this for other methods with similar docstrings